### ISSUE TITLE
Enlarged, Taller Banners on Dragons

### ISSUE THREAD

**Issue by dmdizzy** [AlexModGuy/Ice-and-Fire-Suggestions#1035]
When you place a banner on a dragon, it should be enlarged and the pole made longer, to increase visibility; currently, they're almost impossible to see except from above, and they're so tiny as to be nigh-invisible in relation to the dragon.

**Comment by NeutronForge** [comment]
Yes. Please and better placement. They are in some weird angle on the dragons back
